stdlib/public/Reflection/TypeLowering.cpp

```diff
@@ -789,17 +841,20 @@ class BitMask {
 
 private:
   void andMask(void *maskData, unsigned len, unsigned offset) {
-    assert(offset + len <= size);
+    // Bits beyond the current size are implicitly zero
+    assert(offset < size);
+    unsigned common = std::min(len, size - offset);
     uint8_t *maskBytes = (uint8_t *)maskData;
-    for (unsigned i = 0; i < len; ++i) {
+    for (unsigned i = 0; i < common; ++i) {
       mask[i + offset] &= maskBytes[i];
     }
   }
```

Comment:
Should this zero `mask[i]` from `i=0` to `i=offset-1` and from `i=common+offset` to `i=size`? That would seem consistent with the behaviour of the `andMask()` functions above.